dmc/customer/services/github_service/fake_github_service.rb

```diff
@@ -16,7 +16,7 @@ def fetch_token(code)
     end
 
     def user(oauth_token)
-      FakeUser.new('Aaron Patterson', 'aaron@tenderlove.org')
+      FakeUser.new('Aaron Patterson', 'aaron@tenderlove.org', '')
```

Comment:
To be consistent why do you think about returning an URL like https://avatars0.githubusercontent.com/u/3124?v=3